plugins/techdocs-backend/src/techdocs/stages/publish/local.test.ts

```diff
@@ -37,7 +38,21 @@ const logger = getVoidLogger();
 
 describe('local publisher', () => {
   it('should publish generated documentation dir', async () => {
-    const publisher = new LocalPublish(logger);
+    const testConfig = ConfigReader.fromConfigs([
+      {
+        context: '',
+        data: {
+          backend: {
+            baseUrl: 'http://localhost:7000',
+            listen: {
+              port: 7000,
+            },
+          },
+        },
+      },
+    ]);
```

Comment:
Could this be part of `@backstage/test-utils` package i.e. to get a "default" config for testing purposes?
(Maybe there is, already)

Reply:
No, I'd just make a mock `SingleHostDiscovery` instead.

Reply:
@freben Do you mean something like this -

```
const mockDiscovery = new SingleHostDiscovery(mockInternalBaseUrl, mockInternalBaseUrl)
```

Looks like I can't create a mock `SingleHostDiscovery` like this since the constructor is private. And I have to rely on its static `fromConfig` method.

Reply:
More like `const disc: jest.Mocked<SingleHostDiscovery> = { resolve: jest.fn() };` and then in your tests do `disc.resolve.mockReturnResolvedValueOnce('http://blah')` (all of this typed out without any testing so the names and stuff may be way off)

Reply:
aha! Yes, ofcourse. Thank you for this too.